main.py

```diff
@@ -73,9 +73,9 @@
   # here you can customise the sensor readings by adding extra information
   # or removing readings that you don't want, for example:
   # 
-  #   del readings["temperature"]        # remove the temperature reading
+  #   del reading["temperature"]        # remove the temperature reading
```

Comment:
`readings` got changed to `reading` in this comment. As you make no other changes to main.py, I suggest you undo this and the following line change. Perhaps raise a separate PR if these comments are in fact incorrect?

Reply:
`reading` is correct in this context, but it was just minor cleanup, there's no reason to include it in this PR.